src/routes/open/components/Layout.tsx

```diff
@@ -105,7 +105,7 @@ export const Layout = (props: LayoutProps): React.ReactElement => {
 
   useEffect(() => {
     if (provider) {
-      initContracts()
+      instantiateSafeContracts()
```

Comment:
Missing `then`?, I think it's not needed but just to be sure

Reply:
mmm seems like it's not needed, it was working fine before with `initContracts()`